packages/logger-json/src/index.ts

```
@@ -0,0 +1,184 @@
+        return messageArg;
+      });
+    const finalMessage: Record<string, any> = {
+      ...this.meta,
+      level,
+      time: new Date().toISOString(),
+    };
+    if (this.name) {
+      finalMessage['name'] = this.name;
+    }
+    const extras: any[] = [];
+    for (let messageArg of flattenedMessageArgs) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'messageArg' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)